common/src/region.rs

```diff
@@ -118,7 +128,7 @@ impl Block {
     }
 }
 
-#[derive(Deserialize, Serialize, Copy, Clone, Debug, PartialEq)]
+#[derive(Deserialize, Serialize, Copy, Clone, Debug, JsonSchema, PartialEq)]
 pub struct RegionDefinition {
```

Comment:
Are there any consumers of this API right now?

If so, I'd suggest taking this opportunity to fix the duplicated data (#923) before they get too used to it.

If it's only used for debugging, then it's fine to leave as-is.

Reply:
The only consumer is the downstairs itself.  One downstairs calls anothers repair endpoint.

If we want to fix #923 then I think it might be better to roll that in its own PR as I suspect it will
be a bunch of boilerplate changes (and an upstairs/downstairs version change too possibly).  
I would not want these changes to also include that.

Reply:
Yeah, I realized #923 is going to be more complicated to fix because it's already baked into our `region.json` files, so I'm fine leaving it alone.